integrationTests/vm/testIndexer.go

```diff
@@ -1,368 +0,0 @@
-package vm
-
-import (
-	"bytes"
-	"encoding/json"
-	"fmt"
-	"math/big"
-	"sync"
-	"testing"
-	"time"
-
-	elasticIndexer "github.com/ElrondNetwork/elastic-indexer-go"
-	"github.com/ElrondNetwork/elastic-indexer-go/converters"
-	indexerTypes "github.com/ElrondNetwork/elastic-indexer-go/data"
-	elasticProcessor "github.com/ElrondNetwork/elastic-indexer-go/process"
-	"github.com/ElrondNetwork/elastic-indexer-go/process/accounts"
-	blockProc "github.com/ElrondNetwork/elastic-indexer-go/process/block"
-	"github.com/ElrondNetwork/elastic-indexer-go/process/logsevents"
-	"github.com/ElrondNetwork/elastic-indexer-go/process/miniblocks"
-	"github.com/ElrondNetwork/elastic-indexer-go/process/operations"
-	"github.com/ElrondNetwork/elastic-indexer-go/process/statistics"
-	"github.com/ElrondNetwork/elastic-indexer-go/process/transactions"
-	"github.com/ElrondNetwork/elastic-indexer-go/process/validators"
-	"github.com/ElrondNetwork/elrond-go-core/core"
-	"github.com/ElrondNetwork/elrond-go-core/data"
-	"github.com/ElrondNetwork/elrond-go-core/data/block"
-	outportcore "github.com/ElrondNetwork/elrond-go-core/data/outport"
-	"github.com/ElrondNetwork/elrond-go-core/data/receipt"
-	"github.com/ElrondNetwork/elrond-go-core/data/smartContractResult"
-	"github.com/ElrondNetwork/elrond-go-core/hashing"
-	"github.com/ElrondNetwork/elrond-go-core/marshal"
-	"github.com/ElrondNetwork/elrond-go/integrationTests/mock"
-	"github.com/ElrondNetwork/elrond-go/outport"
-	"github.com/ElrondNetwork/elrond-go/process"
-	"github.com/ElrondNetwork/elrond-go/sharding"
-	"github.com/stretchr/testify/require"
-)
-
-type testIndexer struct {
```

Comment:
suggestion to keep this in a form or another, not necessarily to output data in an indexer 
Reason: with this we test the integration between the processor & outport driver